tssc/step_implementers/shared/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""TODO: doc me
```

Comment:
Any way we can get this resolved prior to merging, @itewk ?

Reply:
fixed.